core/os/file_access.cpp

```diff
@@ -525,6 +525,28 @@ void FileAccess::store_line(const String &p_line) {
 	store_8('\n');
 }
 
+void FileAccess::store_csv_line(const Vector<String> &p_values, const String &p_delim) {
+
+	ERR_FAIL_COND(p_delim.length() != 1);
+
+	String line = "";
+	int size = p_values.size();
+	for (int i = 0; i < size; ++i) {
+		String value = p_values[i];
+
+		if (value.find("\"") != -1 || value.find(p_delim) != -1) {
```

Comment:
I think quotes are also necessary when there are newlines in the value, I found LibreOffice Calc does that

Reply:
Thanks, I added it.